### ISSUE TITLE
Xpath is not not working with the latest geckodriver

### ISSUE THREAD

**Issue by reazuahmed** [mozilla/geckodriver#1480]
Used Java Code: 
driver.findElement(By.xpath("//*[@id=\'main_table\']/span[4]/a")).click();

Error:
*** Element info: {Using=xpath, value=//*[@id='main_table']/span[4]/a}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)

## System

* Version: 
* Platform: 
* Firefox: 
* Selenium:

**Comment by whimboo** [comment]
Could you please attach a [trace-level log] from geckodriver? Read more about reporting actionable bugs in our [contribution guidelines].

[trace-level log]: https://firefox-source-docs.mozilla.org/testing/geckodriver/geckodriver/TraceLogs.html
[contribution guidelines]: https://firefox-source-docs.mozilla.org/testing/geckodriver/geckodriver/Bugs.html

**Comment by whimboo** [comment]
Use the Firefox developer tools for example to retrieve the correct XPath. It can be done by right-clicking the element in the inspector, and selecting `Copy -> XPath` from the context menu.

The right XPath (it will even work with By.ID) is: `//*[@id="u_0_2"]`

**Comment by khajjafari** [comment]
I'm having the same issue. I'm using Firefox 66.0.3. The xpath works on chromdriver but not on geckodriver. I did copy the xpath directly from the firefox developer tools and it is still not working. the only xpath that is working for me is /html. As soon as I add /html/body, it stops working and cannot find the element. I tested on the google home page just to be sure and still the same issue. CSS seems to be working fine though.

![xpathNotWorking](https://user-images.githubusercontent.com/48577893/57260555-fbf25700-7018-11e9-81bc-eb9308744305.png)
